frontend/src/components/ElectionForm/QuickPoll.tsx

```diff
@@ -11,29 +11,22 @@ import { useLocalStorage } from '../../hooks/useLocalStorage';
 import Typography from '@mui/material/Typography';
 import { usePostElection } from '../../hooks/useAPI';
 import { useCookie } from '../../hooks/useCookie';
+import { Election } from '../../../../domain_model/Election.js';
 
 const QuickPoll = ({ authSession }) => {
     const [tempID, setTempID] = useCookie('temp_id', '0')
     const navigate = useNavigate()
     const { error, isPending, makeRequest: postElection } = usePostElection()
-    const onSubmitElection = async (election) => {
-        // calls post election api, throws error if response not ok
-        const newElection = await postElection(
-            {
-                Election: election,
-            })
-        if ((!newElection)) {
-            throw Error("Error submitting election");
-        }
-        localStorage.removeItem('Election')
-        navigate(`/Election/${newElection.election.election_id}`)
-    }
 
-    const QuickPollTemplate = {
+    const QuickPollTemplate: Election = {
```

Comment:
Looks like the quick poll reset every time?

I think this is usually the correct behavior, but for the following flow it would be nice to have it remember the previous info

1. User partially fills out the quick poll, but realizes they want to specify more things
2. User is not logged in, so they click "Login for more settings"
3. When they come back they should be able to continue where they left off 

I see 2 implementation options
1. The QuickPoll component checks if session_state is in the url, and only reads localStorage if that's true (kind of hacky, but I think it would work)
2. Clicking "login for more settings" directs the user to the admin page, and then the admin page will redirect to the login flow if the user isn't logged in. This way they'll be redirected to admin page with more settings after login

Option 2 requires fewer clicks from the user, but seeing the admin page might be confusing to them, not sure which is better

Reply:
QuickPoll doesn't get reset. This input to the useLocalStorage hook is just the default value if it doesn't exist in local storage already.

Reply:
Ok great! Then there are 2 other questions

1. What happens if someone partially fills out the quick poll, and then creates a new election using the nav bar? Will the election creation know to ignore the quick poll info?
2. (nit) Should we have an expiration setup so that people don't to see their partial quick poll days/weeks into the future?

Reply:
hmm, I feel like it should ignore it, but I don't feel too strongly

Everything else looks good so I'll approve. We can create tasks for those items if you think they're worth addressing in the future